eng/pipelines/runtime-staging.yml

```diff
@@ -58,262 +58,262 @@ jobs:
 # iOS/tvOS/Catalyst interp - requires AOT Compilation and Interp flags
 # Build the whole product using Mono and run libraries tests
 #
-- template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
Try to edit the platform-matrix.yml instead and keep just `# WebAssembly on Windows` section. That is usually enough to reduce the CI jobs to just 1 wasm/windows in the runtime-staging.